docs/collection_metadata.rst

```diff
@@ -3,19 +3,30 @@
 Collection-level dependencies
 =============================
 
-When Ansible Builder installs collections into an execution environment, it also installs the dependencies listed by each collection on Galaxy.
+When Ansible Builder installs collections into an execution environment, it also installs their controller-side Python or system package dependencies listed by each collection on Galaxy.
 
-For Ansible Builder to find and install collection dependencies, those dependencies must be defined in one of these files:
+For Ansible Builder to find and install collection dependencies, those dependencies must be defined in files in a collection repository.
 
-* The ``meta/execution-environment.yml`` file containing the Python
-  and/or bindep requirements or referencing other files listing them.
-  The ``.yaml`` extension is also valid on this file.
-* The ``requirements.txt`` file in the root level of the collection.
-* The ``bindep.txt`` file in the root level of the collection.
+.. note::
+
+  If present, the files below must be included in the packaged collection on Galaxy.
+  Ansible Builder cannot install dependencies listed in files that are included in the ``build_ignore`` of a collection, because those files are not uploaded to Galaxy.
```

Comment:
Galaxy is one possible place to store collection artifacts (next to Automation Hub, a local galaxy_ng, the filesystem ...). This should better say something like:
```suggestion
  Ansible Builder cannot install dependencies listed in files that are included in the ``build_ignore`` of a collection, because those files are not included in the collection artifact.
```
or
```suggestion
  Ansible Builder cannot install dependencies listed in files that are included in the ``build_ignore`` of a collection, because those files are not included in the installed collection.
```